SimonImanigirimpuhwe/my-brand-website
# #49 ft(profile): update profile information

#### What does this PR do?

- Update profile information

#### Description of Task to be completed?

- Update user profile information 

- Store updated information to the database

#### How should this be manually tested?

- Login and get to the dashboard

- Then  click on `Edit profile` You should see the form to edit and fill the inputs

#### Any background context you want to provide?

-To be able to edit profile you must first login

#### What are the relevant Trello stories?

- [#49](https://trello.com/c/fYv6hv7k/49-update-profile-information)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A
